.travis.yml

```diff
@@ -123,16 +120,16 @@ jobs:
         - |
           export CONDUIT_TAG=$(. bin/_tag.sh ; clean_head_root_tag)
           echo "CONDUIT_TAG=${CONDUIT_TAG}"
-        - export PROXY_RELEASE=1 BUILD_DEBUG=1 DOCKER_TRACE=1
+        - export BUILD_DEBUG=1 DOCKER_TRACE=1
 
       script:
-        - bin/docker-build
+        - SKIP_TESTS=1 bin/docker-build
```

Comment:
In the previous version of this PR I asked why we're skipping the tests here and you said we're skipping them because the tests have already been run at this point. At least let's add a comment mentioning that here.

I also filed #280 to clean this up, as I don't think we should continue to maintain without-Docker and with-Docker build scripts going forward.